go/vt/vttablet/tabletmanager/vdiff/engine.go

```diff
@@ -221,6 +221,11 @@ func (vde *Engine) addController(row sqltypes.RowNamedValues, options *tabletman
 	globalStats.mu.Lock()
 	defer globalStats.mu.Unlock()
 	globalStats.controllers[ct.id] = ct
+
+	// run() can start a vdiff in pending/started state, so we should init the stats first before starting it.
```

Comment:
This comment is no longer applicable. And we could revert the related changes too. Fine either way IMO.